app/controllers/clients/sessions_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Clients::SessionsController < Devise::SessionsController
+  def destroy
+    session[:intake_id] = nil
```

Comment:
I'd like to know, on logout -- do we clear the entire session? If so, this isn't needed. Maybe we can Tuple and watch the network console? Harmless to keep it.

Reply:
yeah, i actually think maybe this is not needed when we remove the intake_id when someone is logged in. I can also remove this code and leave the test and see if it clears.